documentation/universal_links.md

```diff
@@ -0,0 +1,250 @@
+# Universal Links
+
+- [Universal Links](#universal-links)
+  - [What are Universal Links?](#what-are-universal-links)
+  - [Setting Up an Entitlements File for Universal Links](#setting-up-an-entitlements-file-for-universal-links)
+    - [Steps to Configure the Entitlements File](#steps-to-configure-the-entitlements-file)
+    - [Points to Consider](#points-to-consider)
+  - [Configuring Universal Links for your Storefront](#configuring-universal-links-for-your-storefront)
+  - [Important: The catch-all route `*`](#important-the-catch-all-route-)
+  - [Handling Universal Links in your app](#handling-universal-links-in-your-app)
+  - [Security Considerations](#security-considerations)
+  - [Further Reading and Resources](#further-reading-and-resources)
+
+## What are Universal Links?
+
+Universal Links are an iOS feature which allow apps to open links directly in an app rather than sending the user to a browser. This feature provides a seamless user experience by directly engaging app users with relevant content.
+
+## Setting Up an Entitlements File for Universal Links
+
+To enable Universal Links in your iOS app, you'll need to configure an entitlements file that specifies the associated domains your app is allowed to handle. Here’s how you can set it up:
+
+### Steps to Configure the Entitlements File
+
+1. **Open Your Xcode Project**:
+
```

Comment:
Hey, we're adding a how to guide to shopify dev for authentication.

A `Set up offsite payment handling` might fit nicely into that too?